client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -203,7 +203,7 @@ class PlanningListSubNavComponent extends React.Component<IProps, IState> {
                         </div>
                     )}
 
-                    <ButtonGroup align="end">
+                    <ButtonGroup className = "btnGroup" align="end">
```

Comment:
There is no need for creating a separate button group here, it can go in the same as the firsat dropdown.